docs/contracts/using-for.rst

```diff
@@ -7,27 +7,32 @@ Using For
 *********
 
 The directive ``using A for B;`` can be used to attach
-functions (``A``) as member functions to any type (``B``).
-These functions will receive the object they are called on
+functions (``A``) as operators to user-defined value types
+or as member functions to any type (``B``).
+The member functions receive the object they are called on
 as their first parameter (like the ``self`` variable in Python).
+The operator functions receive operands as parameters.
 
 It is valid either at file level or inside a contract,
 at contract level.
 
 The first part, ``A``, can be one of:
 
-- A list of file-level or library functions (e.g. ``using {f, g, h, L.t} for uint;``) -
-  only those functions will be attached to the type as member functions.
-  Note that private library functions can only be specified when ``using for`` is inside the library.
+- A list of functions, optionally with an operator name assigned (e.g.
+  ``using {f, g as +, h, L.t} for uint;``).
```

Comment:
```suggestion
  ``using {f, g as +, h, L.t} for uint``).
```

or 

```suggestion
  ``using {f, g as +, h, L.t} for uint global;``).
```

Or not sure what to do - it's a minor bug we can fix later if need be - may make things a bit confusing to mix everything together here.

Reply:
Good point. I'll fix that.

Reply:
I now removed the semicolons after such examples.